actionview/Rakefile

```diff
@@ -8,7 +8,7 @@ task :package
 # Run the unit tests
 
 desc "Run all unit tests"
-task test: ["test:template", "test:integration:action_pack", "test:integration:active_record"]
+task test: ["test:template", "test:integration:action_pack", "test:integration:active_record", "test:integration:ujs"]
```

Comment:
👍 to that, we already run Action Cable's integration tests in a separate job task.